api/src/cle/etablissement/etablissementController.ts

```diff
@@ -219,6 +223,59 @@ router.delete("/:id/referents", passport.authenticate("referent", { session: fal
   }
 });
 
+router.post("/", passport.authenticate("referent", { session: false, failWithError: true }), async (req: UserRequest, res) => {
+  try {
+    const { error, value } = Joi.object({
```

Comment:
```suggestion
   const { error, value } = Joi.object<{ uai: string; email: string; refLastName: string; refFirstName: string }>({
```
pour que value soit typé.
je ferait aussi un  alias sur `const { error, value: payload } =` pour que l'on sache à quoi correspond "value" qui est beaucoup trop générique.

Reply:
ca fait pas un peu double emploi un Joi plus un type ?
et TypeScript n'infere pas le type de payload avec le Joi ?

Reply:
Joi ne peux pas faire ça, on est obligé de redonner le type.
ou vous reparle de tout ça en point tech sur la nouvelle archi de typage des routes.